lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Patch looks fine but I'm wondering if we should do this in `api.rb` instead, right before returning the transaction (when returning `authorizePayment`, `getPaymentInfo`, ...)?

That way, we could add a log entry for these originally `UNDEFINED` transactions that we are _fixing_ on the fly, if we need to go back and identify them later on (it would also make it possible to have a config flag I believe).

Not trying to be difficult but to explain where I'm coming from: we still have very little experience with Orbital, and I'm just trying to be pro-active and be ready if (when?) we experience a large outage and we have to go through these transactions manually (more data, logs, configuration knobs we have, easier it will be). But if you think I'm over-thinking it, happy to merge it! 😼